src/styles/User/Login/Login.module.scss

```
@@ -1,54 +1,81 @@
-.login {
-  width: 400px;
-  height: 450px;
-
-  margin: 0 auto;
-  border-style: groove;
-  border-radius: 15px;
-}
-
-.loginBody {
-  display: flex;
-  flex-direction: column;
-}
-
-.loginTitle {
-  width: 84px;
-  margin: 0 auto;
-
-  margin-top: 45px;
-  margin-bottom: 45px;
+@font-face {
+  font-family: 'twayair';
+  src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_tway@1.0/twayair.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
 }
 
-.idInput {
-  width: 280px;
-  height: 35px;
-  margin: 0 auto;
-  margin-bottom: 10px;
-}
+.login {
+  font-family: 'twayair';
+  width: 42em;
+  height: 28rem;
 
-.pwInput {
-  width: 280px;
-  height: 35px;
-  margin: 0 auto;
-  margin-bottom: 25px;
+  margin: 16vh auto;
+  border-style: solid;
+  border-color: #C2F2DC;
+  border-radius: 70px;
+  .body {
+    display: flex;
+    flex-direction: column;
+  }
+  h1 {
+    margin: 0 auto;
+    font-size: 2.5em;
+    margin-top: 45px;
+    margin-bottom: 35px;
+  }
+  .idInput {
+    font-size: 1rem;
+    width: 30rem;
+    height: 3.3rem;
+    margin: 0 auto;
+    margin-bottom: 20px;
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    border-color: #dadada;
```

```suggestion
    border-bottom: 1px solid #dadada;
```
필요한 부분만 생성하는게 필요 없는 부분을 다 지우는 것보다 좋을거같아요

---

오호 이따 다시 적용해서 커밋 보낼게용